airflow/decorators/__init__.py

```diff
@@ -22,6 +22,7 @@
 from airflow.decorators.python_virtualenv import virtualenv_task
 from airflow.decorators.task_group import task_group
 from airflow.models.dag import dag
+from airflow.providers.cncf.kubernetes.decorators.kubernetes import kubernetes_task
```

Comment:
This can’t be done like this here because the user may not have the kubernetes provider installed. You need to use the provider mechanism to declare the decorator. See [development documentation](https://airflow.apache.org/docs/apache-airflow/stable/howto/create-custom-decorator.html) for a more detailed explaination, and how the `@task.docker` decorator is implemented as an example.

Reply:
Removed.